client/index.ts

```diff
@@ -35,7 +35,7 @@ const pluginOptions: PluginOptions = {
 };
 
 const optPath = resolve(cwd(), config);
-let opts = {};
+let opts: any = {};
```

Comment:
Better:
```ts
let opts: Function | Object = null;
```

Reply:
How this line pass the whole tests that you did before?
It's not a new line and look like this error appears also in your master..., right?

Reply:
Which error? TS doesn't enforce typing, so if you don't say nothing to it, it will assume the variable as `any`